rocketpy/simulation/flight.py

```diff
@@ -1422,6 +1429,30 @@ def u_dot(self, t, u, post_processing=False):
             drag_coeff = self.rocket.power_off_drag.get_value_opt(free_stream_mach)
         rho = self.env.density.get_value_opt(z)
         R3 = -0.5 * rho * (free_stream_speed**2) * self.rocket.area * (drag_coeff)
+        for airbrakes in self.rocket.airbrakes:
+            if airbrakes.deployed_level > 0:
+                R3 += (
+                    -0.5
+                    * rho
+                    * (free_stream_speed**2)
+                    * airbrakes.reference_area
+                    * airbrakes.cd(airbrakes.deployed_level, free_stream_mach)
+                )
+                airbrakes.state_history.append(
+                    [
+                        t,
+                        airbrakes.deployed_level,
+                        airbrakes.cd(airbrakes.deployed_level, free_stream_mach),
+                    ]
+                )
+            else:
+                airbrakes.state_history.append(
```

Comment:
I wonder what would happen if you simply forget the if statement and use only the next code lines: 

```python
R3 += (
                    -0.5
                    * rho
                    * (free_stream_speed**2)
                    * airbrakes.reference_area
                    * airbrakes.cd(airbrakes.deployed_level, free_stream_mach)
                )
                airbrakes.state_history.append(
                    [
                        t,
                        airbrakes.deployed_level,
                        airbrakes.cd(airbrakes.deployed_level, free_stream_mach),
                    ]
                )
```

Also, the content inside the append is slightly different from one condition to another. Any particular reasons for that?